client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,66 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
+  static targets = ['total', 'label'];
+  static values = {
+    containerBody: { type: String, default: 'body' },
+    min: { type: Number, default: 0 },
+    find: {
+      type: String,
+      default: '.error-message,.help-critical',
+    },
+    total: { type: Number, default: 0 },
+  };
+
+  declare containerValue: string;
+  declare minValue: number;
+  declare findValue: string;
+  declare totalValue: number;
+
+  declare readonly activeClass: string;
+  declare readonly hasActiveClass: string;
+  declare readonly hasLabelTarget: string;
+  declare readonly hasTotalTarget: string;
+  declare readonly labelTarget: HTMLElement;
+  declare readonly totalTarget: HTMLElement;
+
+  connect() {
+    // console.log("connected aman world");
+    setTimeout(() => {
+      console.log('connected CountController after load. ');
+      this.count();
+    }, 2000);
```

Comment:
Why do you need to wait 2seconds here? It should be OK to run this the moment the controller connects as the DOM will be ready.

Reply:
well yes i was originally, putting it in a hacky manner just to check what's working and what's not, how and why..